test/records/test_models.py

```diff
@@ -1,21 +1,175 @@
 import logging
 
-from app.records.models import Record
+from app.records.models import APIResponse, Record
 from django.test import SimpleTestCase
 
 
 class RecordModelTests(SimpleTestCase):
     maxDiff = None
 
     def setUp(self):
-
         # record structure
         self.source = {
-            "@template": {"details": {}},
+            "@template": {
+                "details": {
+                    "iaid": "C123456",
+                    "title": "Ministry of Defence: Joint Intelligence Bureau and Defence Intelligence Staff: Intelligence Conferences, Committees and Working Parties: Reports and Papers.",
+                    "summaryTitle": "This record has no title",
+                    "dateCovering": "2015-07-31",
+                    "source": "CAT",
+                    "referenceNumber": "LO 2",
+                    "creator": [
+                        "Donop, Stanley Brenton, 1860-1941",
+                        "Lambart, Frederick Rudolph, 1865-1946",
+                        "Thomson, William Montgomerie, 1877-1963",
+                    ],
+                    "formerDepartmentReference": "African No. 355",
+                    "dimensions": "63.5 cm x 68.5 cm",
```

Comment:
We already have test/records/test_catalogue_record_responses.py which show how the attributes function together for a single response. 

Its feels crucial these model tests should be independent of most data in one record.

Ex:  https://github.com/nationalarchives/ds-search/blob/bfefa2a41cd98798d3f6dc95e6deaf4fed7a9f60/test/records/test_models.py#L12-L15

Also when reading it becomes more clearer about the relationships of those attributes
Example:
https://github.com/nationalarchives/ds-search/blob/bfefa2a41cd98798d3f6dc95e6deaf4fed7a9f60/test/records/test_models.py#L77-L104

Reply:
Ok. In that case it would have be easier to just have them skipped to look later, as it would appear that it was intended.

Reply:
@ahosgood I'm happy to get the tests which have been modified or deleted with a PR into this branch if that works for you. And you can focus on the other bits. Let me know.